quality/software-checklist.md

```diff
@@ -5,26 +5,34 @@
 - [ ] Does the software have a descriptive name?
 - [ ] Is there a short high-level description of the software?
 - [ ] Is the purpose of the software clear?
+- [ ] Does the software exactly match its requirements?
 - [ ] Is the targeted audience of the software clear?
-- [ ] Does it (and its dependencies) use OSI approved licenses?
+- [ ] Has the software been tested by members of the target audience in respect of its usability?
+- [ ] Does the software (and its dependencies) use OSI approved licenses?
 - [ ] Is the software under version control?
 - [ ] Is there a website for the software?
+- [ ] Is the software's website mobile friendly to a certain degree? I.e. can it be accessed on a
+  smartphone or tablet without hiding the most important information and features?
+- [ ] Are the user interface design and the software's website mindful of accessibility? I.e. does it
+  consider e.g. a high contrast between colors for colorblind users, are there alternative texts for
+  images that can be read by a screenreader, are texts easily resizeable, ...?  
 - [ ] Does the software have a release mechanism?
 - [ ] Is the software available in packaged format or only sources?
-- [ ] Are maintainer and development status clear, including contact information?
+- [ ] Are maintainer and development status clear, including up to date and accessible contact information?
```

Comment:
Up to date until time of writing but I don't think we can expect all these information to always be up to date.

Reply:
Which scenarios do you have in mind where this is not possible?

Reply:
I mean tools or libraries still being used but not maintained so no contact exists, but it is very likely implied so I will accept it as such.